### ISSUE TITLE
md-input-container should be able to have md-icon-button suffix

### ISSUE THREAD

**Issue by mmalerba** [angular/material2#4382]
currently this causes the placehodler and input text to get misaligned, since the button is taller than the line-height of the input

**Comment by jelbourn** [comment]
Related #2124

**Comment by arlowhite** [comment]
@jelbourn I think this is resolved? `npm run demo-app` scroll down to **Prefix + Suffix / Icon buttons**

**Comment by gedclack** [comment]
Just checked [https://mmalerba-demo4.firebaseapp.com/input](url) and looks good.
@arlowhite but I guess it's not released yet..

**Comment by mmalerba** [comment]
Yep fixed now
